static/js/settings_linkifiers.js

```diff
@@ -105,6 +126,59 @@ exports.build_page = function () {
         });
     });
 
+    $(".admin_filters_table").on("click", ".edit", function (e) {
+        e.preventDefault();
+        e.stopPropagation();
+
+        const btn = $(this);
```

Comment:
We prefer `e.currentTarget` and similar over `this` for this sort of selector, stylistically, though again I note this module seems to already have the style you're using.  Maybe a good follow-up project to bring this module in line with the more recently improved settings pages.

Reply:
Again, I have left this unchanged, since you mentioned that this will be taken in the follow-up project.